tests/lib/rules/no-async-actions.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-async-actions.js');
+const RuleTester = require('eslint').RuleTester;
+
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const message = 'Do not use async actions';
```

Comment:
`message` should be imported from the rule file. See the `no-test-and-then` rule for an example.

Reply:
Done